variables.tf

```diff
@@ -80,6 +80,12 @@ variable "os_disk_size_gb" {
   default     = 50
 }
 
+variable "enable_kube_dashboard" {
+  description = "Enable Kubernetes Dashboard."
+  type        = bool
+  default     = false
```

Comment:
default = true

Reply:
I changed it to `true` even though I'm not sure it's the default behavior if you don't specify it.
Anyway, it's fine for me.